### PR TITLE
feat: new website version

### PR DESCRIPTION
**Pull request by derberg**
Fck that was... 😅 Next year we hire an agency 😄 

- changed link to code of conduct as for the conference we need to follow LF one
- I'm not sure why css changes are done so bad, I added only this:

```css
ul,
ol {
	list-style: none;
	margin: 0;
}

ul li {
	list-style-type: circle;
}
```

instead of:

```css
ul,
ol {
	list-style: none;
	padding: 0;
	margin: 0;
}

ul li,
```

I just wanted nice bullet points and as a result it exploded 🤯 Anyway, works on my local like a charm 😄 

Screenshot 2021-09-10 at 12 27 54
Screenshot 2021-09-10 at 12 28 02

### CODE REVIEW

**Comment by derberg** [comment]
with all due respect dear sonarclound, but I don't care about your opinion here

**Comment by imabp** [comment]
This looks really great.. . @derberg 
Are we going with google groups for mailing list ?

**Comment by derberg** [comment]
@imabp this is a temporary solution, as later I will be sending updates to this google group. Link term, for now considering https://rsvpify.com/pricing/ or native zoom webinar registration.

**Comment by imabp** [comment]
What about having it on Twitch Stream 😏

File: sources/pug/pages/index.pug [hunk 1]
```diff
@@ -105,186 +159,194 @@ block page
...
+      //-       #schedule.card-group-custom.card-group-corporate(role='tablist', aria-multiselectable='false')
+      //-         include ../components/card-content/schedule-item-intro/_schedule-item-header
+      //-         +schedule-intro()
 
-              +card-schedule({
-                parentId: 'schedule',
-                slug: 'paul-taylor',
-                case: 1
-              }).card-corporate
+      //-         +card-schedule({
```
Comment (fmvilas) [inline sources/pug/pages/index.pug:174]: Please don't leave commented code.

Comment (fmvilas) [reply sources/pug/pages/index.pug:174]: :trollface:

**Review by fmvilas — APPROVED** [review]

File: sources/pug/pages/index.pug (hunk 1 above)
Comment (derberg) [reply sources/pug/pages/index.pug:174]: oh man, in my notifications I only saw `Please don't leave commented code.` and boy, you triggered me 😆 

a true :trollface: masterpiece 🙇🏼

**Comment by derberg** [comment]
@imabp yeah, this part we will explore with zoom, we can live stream the event to different platforms, something to consider

**Pull request merged**